src/client/stylesheets/style.less

```diff
@@ -378,3 +378,9 @@ hr.wide {
 		width: 100%;
 	}
 }
+
+@media only screen and(max-width:768px){
+.navbar-header{
+	margin-left:-13px !important;
```

Comment:
Hi. Using `!important` works but I think it's not the ideal way to do. Have a look at this comment
https://github.com/bookbrainz/bookbrainz-site/pull/340#discussion_r366814767

Reply:
@prabalsingh24 
yes ! you are right. 
i have made the necessary changes and i'm about to change the PR,
thank you reviewing it

Reply:
@MonkeyDo 
i have done the changes and i think this would be done with merging .